osgp/protocol-adapter-dlms/osgp-dlms/src/main/java/org/opensmartgridplatform/dlms/services/ObjectConfigService.java

```diff
@@ -371,10 +371,11 @@ protected CosemObject handleValueBasedOnModel(
             originalAttribute.copyWithNewValueAndType(value, ValueType.FIXED_IN_METER);
         return cosemObject.copyWithNewAttribute(newAttribute);
       } else {
-        // If no value was found for this device model, then set the value to Dynamic and log a
-        // warning. The command executor should get the value from the meter.
-        log.warn(
-            "Could not find value for devicemodel {} for {}",
+        // If no value was found for this device model, then set the value to Dynamic and log the
+        // message
+        // The command executor should get the value from the meter.
```

Comment:
`The command executor should get the value from the meter.` This is true for the case we discussed, but could not be true in other situations.